### PR TITLE
fix: dialog ui issues

### LINKED ISSUE #4877: [Bug] Make UI adjustment for change network page for Dashboard
### How to reproduce?

1.Go to Dashboard's Wallet page;
2.Click connect wallet or change wallet to open change network page;

### What happened?
Screen Shot 2021-11-22 at 2 33 45 PM

### What should happen?
Adjust the UI and delete all the plugins below, just keep change wallet card.

### Mask Version:

- [ ] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [x] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by BillySh1**
closes #4877

### CODE REVIEW

**Comment by septs** [comment]
plz rename the pr title, thanks

**Comment by czv6803892** [comment]
>

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/components/shared/WalletStatusBox.tsx [hunk 1]
```diff
@@ -125,7 +129,10 @@ export function WalletStatusBox() {
     }, [openSelectProviderDialog])
 
     return account ? (
-        <section className={classes.currentAccount}>
+        <section
+            className={
+                props.isDashboard ? `${classes.currentAccount} ${classes.dashboardBackground}` : classes.currentAccount
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/WalletStatusBox.tsx:134]: `import classNames from 'classnames'`

Comment (BillySh1) [reply packages/mask/src/components/shared/WalletStatusBox.tsx]: done

**Review by NikoManiac — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**